packages/driver/cypress/e2e/issues/761_2968_3973.cy.js

```diff
@@ -56,7 +56,8 @@ describe('issue #761 - aborted XHRs from previous tests', () => {
 // https://github.com/cypress-io/cypress/issues/3973
 if (Cypress.isBrowser('chrome')) {
   describe('issue #3973 - unloaded xhrs do not fire readystatechange event in chrome >= 71', () => {
-    it('cancels pending requests that are incomplete', () => {
+    // TODO: When Intercepted, aborted XHR requests do not resolve the cy.wait command. Resolved by https://github.com/cypress-io/cypress/issues/24492
```

Comment:
```suggestion
    // TODO: When Intercepted, aborted XHR requests do not resolve the cy.wait command. https://github.com/cypress-io/cypress/issues/24492
```

"Resolved by" makes it sound like the linked issue has fixed the issue (so why is the test skipped?).